reference/config_files/global_conf.rst

```diff
@@ -89,6 +88,169 @@ To list all possible configurations available, run :command:`conan config list`.
+
+All the values will be interpreted by Conan as the result of the python built-in `eval()` function:
+
+.. code-block:: text
+
+    # String
+    tools.microsoft.msbuild:verbosity=Diagnostic
+    # Boolean
+    tools.system.package_manager:sudo=True
+    # Integer
+    tools.microsoft.msbuild:max_cpu_count=2
+    # List of values
+    user.myconf.build:ldflags=["--flag1", "--flag2"]
+    # Dictionary
+    tools.microsoft.msbuildtoolchain:compile_options={"ExceptionHandling": "Async"}
+
+
+Configuration data operators
+----------------------------
+
+Available since: `1.46.0 <https://github.com/conan-io/conan/releases>`_
+
+It's also possible to use some extra operators when you're composing tool configurations in your *global.conf* or
+any of your profiles:
+
+* ``+=`` == ``append``: appends values at the end of the existing value (only for lists).
+* ``=+`` == ``prepend``: puts values at the beginning of the existing value (only for lists).
+* ``=!`` == ``unset``: gets rid of any configuration value.
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [settings]
+    ...
+
+    [conf]
+    # Define the value => ["-f1"]
+    user.myconf.build:flags=["-f1"]
+
+    # Append the value ["-f2"] => ["-f1", "-f2"]
+    user.myconf.build:flags+=["-f2"]
+
+    # Prepend the value ["-f0"] => ["-f0", "-f1", "-f2"]
+    user.myconf.build:flags=+["-f0"]
+
+    # Unset the value
+    user.myconf.build:flags=!
+
+
+Configuration in your profiles
+--------------------------------
+
+Let's see a little bit more complex example trying different configurations coming from the *global.conf* and a simple profile:
+
+.. code-block:: text
+    :caption: *global.conf*
+
+    # Defining several lists
+    user.myconf.build:ldflags=["--flag1 value1"]
+    user.myconf.build:cflags=["--flag1 value1"]
+
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [settings]
+    ...
+
+    [conf]
+    # Appending values into the existing list
+    user.myconf.build:ldflags+=["--flag2 value2"]
+
+    # Unsetting the existing value (it'd be like we define it as an empty value)
+    user.myconf.build:cflags=!
+
+    # Prepending values into the existing list
+    user.myconf.build:ldflags=+["--prefix prefix-value"]
+
+
+Running, for instance, :command:`conan install . -pr myprofile`, the configuration output will be something like:
+
+.. code-block:: bash
+
+    ...
+    Configuration:
+    [settings]
+    [options]
+    [build_requires]
+    [env]
+    [conf]
+    user.myconf.build:cflags=!
+    user.myconf.build:ldflags=['--prefix prefix-value', '--flag1 value1', '--flag2 value2']
+    ...
+
+
+Configuration in your recipes
+-------------------------------
+
+From Conan 1.46, the user interface to manage the configurations in your recipes has been improved. For instance:
+
+.. code-block:: python
+
+    import os
+    from conans import ConanFile
+
+    class Pkg(ConanFile):
+        name = "pkg"
+
+        def package_info(self):
```

Comment:
The example is ok but I think that the available methods like `define`, `get`, `append`, `prepend`, `update`... and their arguments deserve a more detailed documentation